### PR TITLE
Lynn Trickey's Static Site

### PR DESCRIPTION
**Pull request by ltrickey**
# Static Site
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| Did you have to resolve any issues when running the HTML Validator? If so, what were they?  |    A lot of suggestions to always start a section with an h1 - which I didn't always do.  Sometimes I put things in a section simply for styling purposes, like if I wanted to float a photo to the left.  Maybe I'm over-doing it with sections.    |
| Why is it important to consider and use semantic HTML? |  For screen readers and so others who read your html can make sense of the structure of your page, or what you're trying to accomplish.  |
| How did you decide to structure your CSS? |   I tried to group it by section - top/navbar main content section and footer.  That way it makes sense where I'd find the css as it correlates to where the element lives on the page. |
| What was the most challenging piece of this assignment? |  NEVE ENDING TWEAKING AND CHANGING THINGS. |
| Describe one area that you gained more clarity on when completing this assignment | A lot more clarity on positioning items on the page, but still want to know how to account for different screen/browser sizes so my beautiful styling looks good everywhere!  |

### CODE REVIEW

**Comment by kariabancroft** [comment]
# Static Site
## What We're Looking For

| Feature | Feedback    |
| :------------- | :------------- |
|  **Baseline** |   |
|   Appropriate Git Usage	|  Yes 	|
|   Answered comprehension questions	|   Yes	|
|   Page fully loads |Yes |
|   No broken links (regular or images) |Yes |
|   Includes at least 4 pages and styling | Yes|
|  **HTML** |   |
|   Uses the high-level tags for organization: `header, footer, main` | Yes |
|   Appropriately using semantic tags: `section, article`, etc. |Yes |
|   All images include alternate text |Yes - though there is one like `alt=""` |
|  **CSS** |   |
|   Using class and ID names in style declarations |Yes, you're using classes for some similar styling needs and IDs for specific element styling. |
|   Style declarations are DRY  |Yes - I didn't really see an obvious scenario where you were repeating your declarations |
|  **Overall** |  I LOVE YOUR DOG. One thing that's nice to do is open external links in a new tab, so this might be good to do for your twitter, facebook, etc. Really nice job - love that you set up your color palette  |

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans|Pacifico");
```
Comment (sckirk) [inline styles/main.css:1]: awesome job importing fonts from Google's API! I remember this feeling a bit "tricky" the first time. ;)

File: styles/main.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+@import url("https://fonts.googleapis.com/css?family=Open+Sans");
+
+/*MY COLOR PALLET
```
Comment (sckirk) [inline styles/main.css:4]: I'm impressed by your coding style all throughout your 20-ish files. I like that you use comments to describe your rationale and help human readers follow along with your code. I'm imagining your color pallet looks beautiful with the lynn2.jpg image.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <li><a href="hobby-blog.html">COMEDY</a></li>
+          <li><a href="about.html">CONTEXT</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="top">
+        <img src="images/lynn&wesley.jpg" alt="lynn">
+        <section id="sticky-lynn">
```
Comment (sckirk) [inline index.html:29]: Without looking at your site (do you have a link to it?), this id name seemed less intuitive than the others, does sticky-lynn have some meaning I'm missing?

**Review by sckirk — COMMENTED** [review]
Nice work! I'm amazed by all of your great content, fresh images, and ALL of your HTML elements and CSS styling! It is apparent that you took pleasure in crafting a static site that makes you proud. You're doing incredible--keep savoring this learning experience!

File: index.html (hunk 3 above)
Comment (ltrickey) [reply index.html:29]: @sckirk thanks for the feedback!  I don't have the site live yet - want to do some more with it before it goes live.  Originally sticky-lynn was going to be display: sticky so it stuck to the top of the screen when people scrolled, however I trashed that idea later on as it was distracting and didn't really serve a purpose.  I probably should change the name of that id!

**Comment by sckirk** [comment]
That ID name makes sense! :)
If you make this site live--please send me the link; I predict it looks gorgeous!


On Apr 7, 2017, at 12:27 PM, Lynn Trickey <notifications@github.com> wrote:

@ltrickey commented on this pull request.

In index.html:

> +
+    <header>
+      <h1><a href="index.html">She's Trickey</a></h1>
+      <nav>
+        <ul>
+          <li><a href="portfolio.html">CODE</a></li>
+          <li><a href="hobby-blog.html">COMEDY</a></li>
+          <li><a href="about.html">CONTEXT</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="top">
+        lynn
+        <section id="sticky-lynn">
@sckirk thanks for the feedback! I don't have the site live yet - want to do some more with it before it goes live. Originally sticky-lynn was going to be display: sticky so it stuck to the top of the screen when people scrolled, however I trashed that idea later on as it was distracting and didn't really serve a purpose. I probably should change the name of that id!

**Pull request closed without merging**